opal/mca/reachable/netlink/configure.m4

```diff
@@ -74,15 +76,17 @@ AC_DEFUN([OPAL_REACHABLE_NETLINK_CHECK_LIBNL3],[
        save_CPPFLAGS=$CPPFLAGS
        save_LIBS=$LIBS
 
-       $1="-I/usr/include/libnl3"
-       CPPFLAGS="$$1 $CPPFLAGS"
-       AC_MSG_CHECKING([for /usr/include/libnl3])
-       AS_IF([test -d "/usr/include/libnl3"],
-             [AC_MSG_RESULT([present])
-              AC_CHECK_HEADER(
-                [netlink/version.h],
-                [AC_COMPILE_IFELSE(
-                    [AC_LANG_PROGRAM([[
+       AS_IF([test $opal_libnl_version -eq 1],
```

Comment:
i am not sure about what you mean.
this macro is used to find out what we can build (e.g. lib plus devel headers are required)
whereas other macro is used to find out what third party libs are linked against (no devel header required)
all i can think about is move and rename the macro (and maybe cache the result) to config/opal_check_libnl.m4